src/Nethermind/Nethermind.Network/ProtocolValidator.cs

```diff
@@ -68,7 +68,7 @@ private bool ValidateEthProtocol(ISession session, ProtocolInitializedEventArgs
 
             if (_forkInfo.ValidateForkId(syncPeerArgs.ForkId.Value, _blockTree.Head?.Header) != ValidationResult.Valid)
             {
-                return Disconnect(session, DisconnectReason.InvalidForkId, CompatibilityValidationType.InvalidForkId, "invalid fork id");
+                return Disconnect(session, DisconnectReason.InvalidForkId, CompatibilityValidationType.InvalidForkId, $"invalid network id {syncPeerArgs.NetworkId} fork id {syncPeerArgs.ForkId.Value}");
```

Comment:
would be good to add `ValidationResult` to the message

Reply:
Done